components/homme/cmake/machineFiles/anlworkstation.cmake

```diff
@@ -0,0 +1,25 @@
+# anlworkstation
```

Comment:
Should this be part of this PR?

Reply:
This change was required to get HOMME tests building/running on anlworkstation. I can move it to a different PR if you want.

Reply:
Its fine here.  Just add a mention to the PR description.

Reply:
ok, done